Monika After Story/game/script-topics.rpy

```diff
@@ -4692,3 +4692,342 @@ label monika_100k:
     else:
         m 1m "Almost~"
     return
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_timeconcern",category=['Advice'],prompt="Is everything okay?",random=True))
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_timeconcern",category=['Advice'],prompt="Is everything okay?",random=True))
+
+
+default persistent.timeconcern = 0
+default persistent.timeconcerngraveyard = 0
+default persistent.timeconcernclose = False
```

Comment:
Can you adjust the `persistent` variable names to be consistent with the styleguide? I've linked it in #915 .